scripts/generate_schemas.py

```diff
@@ -2,7 +2,7 @@
 """Generate JSON schemas from the frameworks questions content.
 
 Usage:
-    generate-schemas.py --output-path=<output_path>
+    generate_schemas.py --output-path=<output_path>
```

Comment:
Let's move the functions out and keep the script as an entry point. It should make it more obvious when we're testing the generation logic as opposed to the script itself.